libraries/AP_HAL_SITL/SITL_State.h

```diff
@@ -195,6 +195,11 @@ class HALSITL::SITL_State {
     VectorN<readings_wind,wind_buffer_length> buffer_wind_2;
     uint32_t time_delta_wind;
     uint32_t delayed_time_wind;
+    enum wind_start_delay_type {
+        WIND_DELAY_INIT = 0,
+        WIND_DELAY_WAIT,
+        WIND_DELAY_OVER
+    };
     uint8_t wind_start_delay;
```

Comment:
If you still want to keep this, then this variable should have type `wind_start_delay_type`.